### PR TITLE
Gmaps removal

### PR DESCRIPTION
**Pull request by kingstuffy**
Moves project into new dev scaffold.
Removes GMaps from project.

### CODE REVIEW

**Review by sc0ttj — COMMENTED** [review]
Looks good to me, nice work! Chris can confirm... 

.. if you prefer, you *could* get rid of the progressiveEnhancement class and move that stuff into app.js (it was dumb of me to make it that way..)

**Comment by kingstuffy** [comment]
@sc0ttj Thanks. I'll do just that

File: Gruntfile.js [hunk 1]
```diff
@@ -1,38 +1,40 @@
-/*jshint es3: false */
+/* jshint es3: false */
```
Comment (eagerterrier) [inline Gruntfile.js]: this makes me super happy... 

Thanks! hahaha

File: Gruntfile.js [hunk 2]
```diff
@@ -1,38 +1,40 @@
...
 module.exports = function (grunt) {
+    const path = require('path');
+    const os = require('os');
+    const config = {
+        pkg: grunt.file.readJSON('package.json'),
+        config: grunt.file.readJSON('config.json'),
+        translations: grunt.file.readJSON('node_modules/@bbc/news-vj-ws-config/config.json'),
+        env: getEnvData(grunt.file.readJSON('config.json')),
```
Comment (eagerterrier) [inline Gruntfile.js]: we just put `grunt.file.readJSON('config.json')` two lines above. Can we set as a constant before the `config` object declaration?

File: Gruntfile.js [hunk 3]
```diff
@@ -1,38 +1,40 @@
...
-            }
+    function getEnvData(config) {
+        let env = {
+            local: {
+                domain: 'http://www.local.bbc.co.uk:1031',
+                domainStatic: 'http://static.local.bbc.co.uk:1031',
+            },
         };
-        var environmentFilePath = path.join(__dirname, '../../env.json');
+        const environmentFilePath = path.join(os.homedir(), config.envJsonRelativeToHomeDirectory);
```
Comment (eagerterrier) [inline Gruntfile.js]: I'm code reviewing this blind, so I may be way off the mark here, but...

`config.envJsonRelativeToHomeDirectory` doesn't appear to be set.

`config` seems to be the object that contains _another_ `config`, so the correct reference _seems_ to be `config.config.envJsonRelativeToHomeDirectory`

File: source/js/app.js [hunk 4]
```diff
@@ -1,132 +1,144 @@
...
         }
+    }
 
-        function setAutopauseEvent(videoIndex) {
-            $.on('video-playing-' + videoSelectors[videoIndex], function () {
-                pauseVideo(videoIndex);
-            });
-        }
+    function setAutopauseEvent(videoIndex) {
+        $.on(`video-playing-${videoSelectors[videoIndex]}`, () => {
```
Comment (eagerterrier) [inline source/js/app.js]: 👍

**Review by eagerterrier — COMMENTED** [review]
nice. Please test that it works thoroughly, as I couldn't do this.

Also, maybe think about getting rid of the `for` loops with ES6 equivalents where possible if you have time. Otherwise, all good. Big commit!

File: Gruntfile.js (hunk 2 above)
Comment (kingstuffy) [reply Gruntfile.js]: I'll do that. We probably need to change this in the dev scaffold too.

File: Gruntfile.js (hunk 3 above)
Comment (kingstuffy) [reply Gruntfile.js]: The `config` is passed into the function and it's populated from the `config.json` in the root directory which has the `envJsonRelativeToHomeDirectory` property, so it's fine.

**Comment by ChrisBAshton** [comment]
Looks good to me, @kingstuffy - I've had a look locally and it seems to work as expected.

I had to fix the node aws client issue, opened a PR here if you'd like to review/merge: https://github.com/BBCVisualJournalism/newsspec_12799/pull/9

If we deploy with Sling then we'll have a new include path so we can always swap back to the old include if the new one is broken for some reason. I vote we merge and give it a go!

**Pull request merged**